deye-inverter-mqtt.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
Thanks for sharing this file. However I'm very much reluctant to add it to the project. If I do this, then very likely I will also have to support it and answer potential questions in the future. I really do not want to do this. I'd rather prefer to add a link to your fork, so if someone is interested in this deployment model, then they can follow. WDYT?

Reply:
Then decline, All fine. Linking to my fork wouldn't be the best choice, because over time this will distinct the two repositories. 

Thanks for the hint with the EnvironmentFile. Didn't knew that, but running now :-)